irohad/consensus/yac/messages.hpp

```diff
@@ -47,11 +48,10 @@ namespace iroha {
        * All nodes deals on some solution
        */
       struct CommitMessage {
-        CommitMessage() = default;
+        CommitMessage() = delete;
```

Comment:
It is not necessary to explicitly delete default constructor since another constructor is defined.